packages/frontend/src/views/create-event.vue

```diff
@@ -10,6 +10,19 @@
             <label for="event-name" class="block py-2 text-xs font-bold tracking-wide text-white uppercase">Event name</label>
             <input type="text" id="event-name" name="event-name" class="appearance-none input" placeholder="LOVE YOUR SELF">
           </div>
+          <div class="grid overflow-x-auto col-span-4 grid-rows-1 gap-y-4 gap-x-6 md:grid-cols-2">
+            <div class="input-box">
+              <label for="event-description-box" class="block py-2 text-xs font-bold tracking-wide text-white uppercase">Description</label>
+              <span
+                class="textarea"
+                role="textbox" contenteditable="true"
+                @input="updateMarkdown" />
```

Comment:
We should store raw string as a variable, what I'm seeing currently is that the text from user is being compiled to markdown, and *that's it*, it doesn't store the *raw* input from user anywhere, you should create a reference variable to store that raw text somewhere to use

also look at [this](https://stackoverflow.com/questions/19030742/difference-between-innertext-innerhtml-and-value) for difference of `innerHTML`, `innerText`, and `value`